presidio-anonymizer/presidio_anonymizer/anonymizers/validators.py

```diff
@@ -0,0 +1,16 @@
+from presidio_anonymizer.entities import InvalidParamException
+
+
+def validate_parameter(
+    parameter_value, parameter_name: str, parameter_type: type
+) -> None:
+    """Validate an anonymizer parameter.
+
+    Both validate a parameter exists and that it is an instance of the
+    parameter_type. Otherwise, raise the appropriate InvalidParamException
+    with the parameter_name as content.
+    """
+    if parameter_value is None:
+        raise InvalidParamException(f"Expected parameter {parameter_name}")
+    if not isinstance(parameter_value, parameter_type):
+        raise InvalidParamException(f"Invalid parameter value for {parameter_name}")
```

Comment:
@omri374 I need your input on the text here

Reply:
Still? it shows as outdated

Reply:
Yes please

Reply:
Maybe add more context on the second one? Something like
```python
f"Invalid parameter value for {parameter_name}. Expecting {parameter_type}  but got {type(parameter_value)}"
```

Reply:
@omri374 Sounds good. It will however expose the internal implementation: "Expecting 'str'  but got 'bool'".

Reply:
we can stranslate the internal implemetation to the user friendly names (str to string etc)

Reply:
Sounds good, in case we don't have value in the translation map, return the general error above, agreed?